README.md

```diff
@@ -122,7 +139,8 @@ can still be sent.
 The `Request` class is responsible for streaming the incoming request body
 and contains meta data which was parsed from the request headers.
 
-It implements the `ReadableStreamInterface`.
+It implements the `ReadableStreamInterface`. The body data will be emitted via
+the `data` and `end` events. Checkout the example in the [Usage](#usage) for more details.
```

Comment:
I'm undecided about this, currently. Shouldn't the example be better located here and perhaps a link from above instead?

Also, what if the body contains invalid data (content-length mismatch or invalid transfer encoding)? Should probably be added to the example?